frontend/src/components/MoviePageJoker.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import MoviePage from "./MoviePage";
+import movieList from "../DataBaseMovie";
+
+const MoviePageJoker = () => (
+  <div className="MoviePageJoker">
```

Comment:
Change your className or erase it (doesn't return to any CSS), thx:)

Reply:
children ClassName deleted